themes/default/content/templates/serverless-application/azure/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Azure Serverless Application"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The Azure Serverless Template makes it easy to deploy a serverless application on Azure with Pulumi, Azure Functions, and Azure Blob Storage.
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a serverless application on Azure with Pulumi, Azure Functions, and Azure Blob Storage.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: serverless-azure
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Azure
```

Comment:
These links get used on the right-hand side and are the full names -- "Amazon Web Services", "Microsoft Azure", "Google Cloud Platform":

```suggestion
  name: Microsoft Azure
```